### ISSUE TITLE
check dinamically checked options into checklist

### ISSUE THREAD

**Issue by giorgio84** [vitalets/x-editable#670]
Hy guys,

I have an answer with the checklist component. Basically a I get a table od Player and every Player has one or more roles (like forward, goalkeeper,ecc...). When I display my table, I would like to see in the roles column all the roles checked. How can I do that for to provide dinamically for every player checked option? I tried to work with the 'value' attribute but I receive always Empty list. 

<a href="#" id="options" data-type="checklist" data-pk="1" data-url="/post" data-title="Select options"></a>

```
<script>
$(function(){
$('#options').editable({
value: [2, 3],
source: [
{value: 1, text: 'option1'},
{value: 2, text: 'option2'},
{value: 3, text: 'option3'}
]
});
});
</script>
```

Basically I will need to substitute value attribute with a function like this:

`...
value: function(){....},
....`

Could someone help me to solve my problem?

Thanks in advance.
